lms/templates/certificates/_accomplishment-footer.html

```diff
@@ -0,0 +1,21 @@
+<%namespace file='/theme-variables.html' import="get_certificates_settings" />
+
+<div class="wrapper-footer">
+
+    <footer class="footer-app" role="contentinfo" id="company-info">
+        <div class="footer-app-copyright">
+            <p class="copy copy-micro">${get_certificates_settings()['footer_copyright_text']}</p>
+        </div>
+        <nav class="footer-app-nav">
+            <ul class="list list-legal">
+                <li class="nav-item">
+                    <a class="action btn btn-small btn-link" href="${get_certificates_settings()['footer_tos_url']}">${company_tos_urltext}</a>
```

Comment:
We should hide the link if the value is unset.

Reply:
Changed. Pushing soon.